pylab/core/models.py

```diff
@@ -56,3 +56,13 @@ def __str__(self):
 
     def get_absolute_url(self):
         return reverse('event-details', args=[self.starts.year, self.starts.month, self.starts.day, self.slug])
+
+
+class Attendance(models.Model):
+    attendee = models.ForeignKey(User)
+    event = models.ForeignKey(Event)
+    responce = models.PositiveSmallIntegerField(choices=(
+        (0, _('No')),
+        (1, _('Yes')),
+        (2, _('Maybe'))
+    ), default=2)
```

Comment:
Follow [Django coding style](https://docs.djangoproject.com/en/1.8/internals/contributing/writing-code/coding-style/#model-style):

> If choices is defined for a given model field, define each choice as a tuple of tuples, with an all-uppercase name as a class attribute on the model. Example:
> 
> ```python
> class MyModel(models.Model):
>   DIRECTION_UP = 'U'
>   DIRECTION_DOWN = 'D'
>   DIRECTION_CHOICES = (
>       (DIRECTION_UP, 'Up'),
>       (DIRECTION_DOWN, 'Down'),
>   )
> ````

Or at least follow [PEP-8](https://www.python.org/dev/peps/pep-0008/#a-foolish-consistency-is-the-hobgoblin-of-little-minds):

> A style guide is about consistency. Consistency with this style guide is important. Consistency within a project is more important. Consistency within one module or function is most important.

And try to be consistent at least [within one module](https://github.com/python-dirbtuves/website/blob/Attendance/pylab/core/models.py#L26-L35).